src/app/components/forms/background/AppFormBackground.vue

```diff
@@ -0,0 +1,97 @@
+<script lang="ts" setup>
```

Comment:
let's also put this functionality into an AppBackgroundSelector component that should work with v-model. And then we should wrap this as AppFormControlBackground and it should live in the _common/form-vue folder since that's where we put all our form stuff